src/vivarium_gates_iv_iron/components/hemoglobin.py

```diff
@@ -36,60 +35,35 @@ def setup(self, builder: Builder):
         ]
         # load data
         # TODO: do this s/location/country in the loader?
-        mean = builder.data.load(data_keys.HEMOGLOBIN.MEAN).rename(
-            columns={"location": "country"}
-        )
-        stddev = builder.data.load(data_keys.HEMOGLOBIN.STANDARD_DEVIATION).rename(
-            columns={"location": "country"}
-        )
-        self.location_weights = builder.data.load(
-            data_keys.POPULATION.PLW_LOCATION_WEIGHTS
-        ).rename(columns={"location": "country"})
-        index_columns = [
-            "country",
-            "sex",
-            "age_start",
-            "age_end",
-            "year_start",
-            "year_end",
-        ]
+        mean = builder.data.load(data_keys.HEMOGLOBIN.MEAN).rename(columns={"location": "country"})
+        stddev = builder.data.load(data_keys.HEMOGLOBIN.STANDARD_DEVIATION).rename(columns={"location": "country"})
+        self.location_weights = self._get_location_weights(builder)
+        index_columns = ["country", "sex", "age_start", 'age_end', 'year_start', 'year_end']
         mean = mean.set_index(index_columns)["value"].rename("mean")
         stddev = stddev.set_index(index_columns)["value"].rename("stddev")
         distribution_parameters = pd.concat([mean, stddev], axis=1).reset_index()
-        self.distribution_parameters = builder.value.register_value_producer(
-            "hemoglobin.exposure_parameters",
-            source=builder.lookup.build_table(
-                distribution_parameters,
-                key_columns=["sex", "country"],
-                parameter_columns=["age", "year"],
-            ),
-            requires_columns=["age", "sex", "country"],
-        )
+        self.distribution_parameters = builder.value.register_value_producer("hemoglobin.exposure_parameters",
```

Comment:
Why did you change this formatting? It was so much more readable before.
Same for the blocks below.

Reply:
This is an artifact of merge conflict resolution.